src/nodes/display/ColorAdjustmentNode.js

```diff
@@ -86,7 +86,7 @@ export const saturation = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.SA
 export const vibrance = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.VIBRANCE );
 export const hue = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.HUE );
 
-export const lumaCoeffs = vec3( 0.2125, 0.7154, 0.0721 );
+export const lumaCoeffs = vec3( 0.2126729, 0.7151522, 0.0721750 ); // // assumes rgb is in linear color space with sRGB primaries and D65 white point
```

Comment:
Looking at https://en.wikipedia.org/wiki/Rec._709#Luma_coefficients, it seems `0.2125, 0.7154, 0.0721` are actually the more recent luma coefficients. They are based on BT.709-1.

Should we update the formula in `common.js`/ `WebGLRenderer` instead?

Reply:
Luma is not the correct term. Luma is computed from components in sRGB color space -- not linear sRGB color space.

https://en.wikipedia.org/wiki/Relative_luminance

Reply:
I don't think Luma is the correct term. Luma is computed from components in sRGB color space -- not linear sRGB color space.

https://en.wikipedia.org/wiki/Relative_luminance

Reply:
I think it's more about consistency than correctness for our purposes – I've seen the 4-digit coefficients `0.2126, 0.7152, 0.0722` most often, that's closer to what we've historically used, Blender uses them, and that's also what the ASC CDL requires... I would be inclined not to update to the newer BT.709-1 coefficients until we observe other software relevant to our users doing the same.

Ideally, this should be color space dependent, and could throw an error for anything but Linear-sRGB and sRGB. 

```javascript
const luminanceCoeffs = ColorManagement.getLuminanceCoefficients(
  ColorManagement.workingColorspace
);
```

My understanding would be that the coefficients are applicable to both sRGB and Linear-sRGB components, but produce different results: luma (for sRGB components) and relative luminance (for Linear-sRGB components)

Reply:
> I think it's more about consistency than correctness for our purposes – I've seen the 4-digit coefficients 0.2126, 0.7152, 0.0722 most often, that's closer to what we've historically used, Blender uses them, and that's also what the ASC CDL requires... I would be inclined not to update to the newer BT.709-1 coefficients until we observe other software relevant to our users doing the same.

Okay, in this case I merge so we are consistent with `WebGLRenderer`.

Reply:
>the coefficients should be color space dependent

Yes, that is why I included the comment in `common.glsl.js`:
```glsl
float luminance( const in vec3 rgb ) {

	// assumes rgb is in linear color space with sRGB primaries and D65 white point

	const vec3 weights = vec3( 0.2126729, 0.7151522, 0.0721750 );

	return dot( weights, rgb );

}
```
>I could look at this later

Great! These coefficients are simply the 2nd row in the conversion matrix from Linear-sRGB to XYZ color space, assuming a D65 white point.

Reply:
BTW: There is a shader which is in used by the unreal bloom pass which relies on even different values:

https://github.com/mrdoob/three.js/blob/b99962244ebb222a4a803f4f8b9284eb8030b170/examples/jsm/shaders/LuminosityHighPassShader.js#L52-L54

These seems to be Rec. 601 luma luma coefficients. Would it make sense to change the code to the following so 709 is used? Related https://en.wikipedia.org/wiki/Luma_(video)#Rec._601_luma_versus_Rec._709_luma_coefficients
```js
const v = luminance( texel.xyz );
```

Reply:
Sounds good! Follow-up PR:

- https://github.com/mrdoob/three.js/pull/28880